tests/components/todoist/test_calendar.py

```diff
@@ -115,6 +115,30 @@ async def test_update_entity_for_custom_project_with_labels_on(
     assert state.state == "on"
 
 
+@patch("homeassistant.components.todoist.calendar.TodoistAPIAsync")
```

Comment:
I wasn't 100% sure how to verify a failed update, but this appears to simulate what would happen if an update on setup failed.  If you have suggestions or examples that might be better let me know.